source/pug/views/home.pug

```diff
@@ -5,36 +5,46 @@ block masterParams
 
 include ../partials/pulse-projects.pug
 
-block heroGuts
-  .row
-    .col-md-12.col-lg-6.py-5
-      .mb-4: h1.h1-white Be a champion of a healthy internet
-      p.lead-black Billions of people depend on a <a href="/about">healthy internet</a>. Our network helps keep this ecosystem open, secure, and accessible to all.
-
 block content
   .container
+    .row.mt-5.d-flex.align-items-center
+      .col-md-6
+        h1.h1-white India's National Registration System
+        p.lead-black.mb-4 Join Mozilla's Executive Chairwoman, Mitchell Baker in signing an open letter to the Justice Srikrishna Committee highlighting our main concerns.
+        a.btn.btn-normal(href=`https://foundation.mozilla.org/campaigns/aadhaar/`) Take Action
+      .col-md-6.hidden-sm-down
+        img(src=`/_images/homepage/phone-bar.jpg`)
     .row
-      .col.my-5
+      .col.my-4
         hr.hr-gradient.mt-0
-        h4.h2-headings-white Latest from the Network
+        h4.h2-headings-white Latest news
     #home-news
     .row
       .col.text-center.mb-5
         a.btn.btn-ghost(href=`/news`) View more news
 
     .row
-      .col.my-5
+      .col.mb-4
         hr.hr-gradient
-        h4.h2-headings-white Network Leaders
-    #featured-people-box.mb-sm-5
-    .row
-      .col.text-center.mb-5
-        a.btn.btn-ghost(href=`/people`) Meet More Leaders
+        h4.h2-headings-white Happening now
+    #home-highlights
 
     .row
-      .col.mb-5
+      .col.mb-2
+        hr.hr-gradient
+    .row
+      .col-md-6
+        h4.h2-headings-white Trending projects
+      .col-md-6.d-flex.justify-content-end
+        a.cta-link(href=`https://www.mozillapulse.org/add`) Add your project
+    .row
+      .col
+        .pulse-project-list(for="", size="6", checked="true")
```

Comment:
Let's use the same cropping behavior that the Pulse client uses so we don't get images that are too tall. See https://www.mozillapulse.org/entry/588 vs what shows up on the review app.